front/app/containers/Admin/sideBar/MenuItem.tsx

```diff
@@ -114,7 +118,13 @@ const MenuItem = ({ navItem }: Props) => {
   const isItemDisabled =
     isProjectProposalsEnabled && navItem.name === 'initiatives';
 
-  if (!featuresEnabled || !hasPermission) return null;
+  if (navItem.name === 'reporting') {
+    if (!isSuperAdmin(user) && (!featuresEnabled || !hasPermission)) {
+      return null;
+    }
+  } else {
+    if (!featuresEnabled || !hasPermission) return null;
+  }
```

Comment:
Maybe we could use something like:
```
  let hasPermission = usePermission({
    action: 'access',
    item: { type: 'route', path: navItem.link },
  }) || (navItem.name === 'reporting' && isSuperAdmin(user));

  // ...
  
  if (!featuresEnabled || !hasPermission) return null;
```
...or not 🙂 

However, I would add a comment to explain why we make an exception for reporting.

Reply:
I don't think that logic works- if `featuresEnabled === false`, the super admin should still have access. In your snippet it will `return null`

Reply:
But indeed will add an extra comment 👍

Reply:
Ah yeah, right. Went too fast 🗑️